packages/docs-reanimated/src/examples/RelativeCoords.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import { View, Text, StyleSheet } from 'react-native';
+import Animated, {
+  useAnimatedRef,
+  getRelativeCoords,
+} from 'react-native-reanimated';
+import { Gesture, GestureDetector } from 'react-native-gesture-handler';
+
+const RelativeCoords = () => {
+  const animatedRef = useAnimatedRef();
+  const [coords, setCoords] = useState({ x: 0, y: 0 });
+  const [boxCoords, setBoxCoords] = useState({ x: 0, y: 0 });
```

Comment:
These names are a bit confusing 🤔 Can you rename them?

Reply:
Probably, when getRelativeCoords will be working, the other (not-relative) coords will be unneccessary. I mainly put them to show the difference, and to catch whats wrong.